packages/jest-runner/errorUtils.test.ts

```diff
@@ -17,10 +17,14 @@
 import {
 	removeTopFrames,
 	removeBottomFrames,
-	cleanupJestRunner,
+	cleanupJestError,
+	cleanupJestRunnerStack,
+	removeTopFramesFromError,
+	removeBottomFramesFromError,
 } from "./errorUtils";
 
 describe("ErrorUtils", () => {
+	const error = new Error();
 	const stack = `Error:
     at /home/norbert/Code-Intelligence/jazzer.js/examples/jest_integration/integration.fuzz.js:27:3
```

Comment:
Despite this being a only test, hard-coded elements like this should be avoided. If we mock a stack trace in the first place would it make sense to "anonymize" the paths or use a more generic one instead?

Reply:
Oh, yes, the path sneaked in again after changing the example stack, removed.
The stack was hard coded here, as asserting the concrete strings seems more understandable than abstract checks on left size or lines.